### PR TITLE
Fix: Pacstall not removing `.deb` packages

### PR DESCRIPTION
**Pull request by NarutoXY**
Where I fcked up earlier:
* Not sourcing LOGDIR
* Not copying pacscript

What I changed: fixed my nasty code. Changed my editor setting to expand tabs lmao and also source log before sourcing pacscript in remove.sh and copy pacscript in install-local.sh

### CODE REVIEW

**Comment by Henryws** [comment]
```bash
➜  ~ pacstall -I brave-browser-beta
Do you want to view the pacscript first [y/N] n
Do you want to edit the pacscript [y/N] n
[+] INFO: Sourcing pacscript
[+] INFO: URL exists
[+] INFO: Installing dependencies
[+] INFO: Retrieving packages
[+] INFO:
Initializing download: https://github.com/brave/brave-browser/releases/download/v1.27.81/brave-browser-beta_1.27.81_amd64.deb
File size: 86406968 bytes
No state file, cannot resume!
Reading package lists... Done
Building dependency tree
Reading state information... Done
Note, selecting 'brave-browser-beta' instead of './brave-browser-beta_1.27.81_amd64.deb'
brave-browser-beta is already the newest version (1.27.81).
0 upgraded, 0 newly installed, 0 to remove and 11 not upgraded.
➜  ~ pacstall -R brave-browser-beta
/usr/share/pacstall/scripts/remove.sh: line 8: /var/cache/pacstall/brave-browser-beta//brave-browser-beta.pacscript: No such file or directory
[!] ERROR: brave-browser-beta is not installed or not properly symlinked
➜  ~
```

**Comment by NarutoXY** [comment]
It works on my machine. Have you changed it to my fix branch??

**Comment by Henryws** [comment]
> It works on my machine. Have you changed it to my fix branch??

yes

File: misc/scripts/install-local.sh [hunk 1]
```diff
@@ -265,8 +265,13 @@ case "$url" in
 		hashcheck "${url##*/}"    
 		sudo apt install -y -f ./"${url##*/}" 2>/dev/null
 		if [[ $? -eq 0 ]]; then
-			loggingMeta
+			log
 			exit 0
```
Comment (D-Brox) [inline misc/scripts/install-local.sh:269]: `exit` comes aftewards

**Review by D-Brox — CHANGES_REQUESTED** [review]

File: misc/scripts/remove.sh [hunk 2]
```diff
@@ -19,20 +33,6 @@ case "$url" in
 	*)
 		cd "$STOWDIR" || (sudo mkdir -p "$STOWDIR"; cd "$STOWDIR")
```
Comment (D-Brox) [inline misc/scripts/remove.sh:34]: This should be in line 7

**Review by D-Brox — CHANGES_REQUESTED** [review]

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Disable expand tabs

**Review by Henryws — APPROVED** [review]
👍🏻

**Review by D-Brox — APPROVED** [review]

**Pull request merged**